src/actions/MapActions.js

```diff
@@ -9,10 +9,11 @@ import {
 } from './types';
 
 
-export const initializeMap = () => {
+export const initializeMap = (region) => {
 
     return {
-        type: INITIALIZE_MAP
+        type: INITIALIZE_MAP,
+        payload: region
```

Comment:
When we're initializing the map, pass along the `region` if we have it.  (For situations where we're rehydrating we'll have a region to start with. For fresh loads of the app, we won't so we'll pass along undefined).

Reply:
You may of already explained this already. But, what's your thought on what we are using for region? current device coordinates? inquired region through search?